dispatcher/dispatcher.go

```diff
@@ -16,28 +17,39 @@ var dispatchers roster.Roster
 // DispatcherInitializationFunc is a function used to initialize an implementation of the `webhookd.WebhookDispatcher` interface.
 type DispatcherInitializationFunc func(ctx context.Context, uri string) (webhookd.WebhookDispatcher, error)
 
+// ctxUrl is a `context.Context` key used to store the URL that a `HTTPDispatcher` instance is configured to send messages to.
+type ctxUrl struct{}
+
+// ctxMethod is a `context.Context` key used to store the HTTP method that a `HTTPDispatcher` instance is configured to use.
+type ctxMethod struct{}
```

Comment:
This should not be a "global" variable inside the `dispatcher` package but rather a query parameter or the entire URI passed to the `NewHTTPDispatcher` (or `NewDispatcher`) method so that these details can be specified in the configuration file as well as supporting multiple "HTTP" dispatchers with different endpoints.

Reply:
Removed as per suggestion